packages/inference/src/snippets/templates/js/huggingface.js/basic.jinja

```diff
@@ -0,0 +1,11 @@
+import { InferenceClient } from "@huggingface/inference";
+
+const client = new InferenceClient("{{ accessToken }}");
+
+const output = await client.{{ methodName }}({
+	model: "{{ model.id }}",
+	inputs: {{ inputs.asObj.inputs }},
+	provider: "{{ provider }}",
+});
+
+console.log(output);
```

Comment:
Following my remark about InferenceClient types - applies to all `huggingface.js` snippets I think
```suggestion
import { {{methodName}} } from "@huggingface/inference";

const output = await {{ methodName }}({
	model: "{{ model.id }}",
	inputs: {{ inputs.asObj.inputs }},
	provider: "{{ provider }}",
	accessToken: "{{ accessToken }}"
});

console.log(output);
```

Reply:
opened https://github.com/huggingface/huggingface.js/issues/1294 as a follow-up issue